install/linux/usr/share/odemis/sim/fastem-sim-asm.odm.yaml

```diff
@@ -319,19 +324,22 @@ FASTEM-sim: {
 "Diagnostic Camera": {
     class: simcam.Camera,
     role: diagnostic-ccd,
+    dependencies: {"focus": "Sample Stage in carrier coordinates"},
     init: {
         image: "../acq/align/test/multiprobe01.tiff",
     },
     properties: {
         exposureTime: 0.005, # s
     },
     metadata: {
-        # Factory calibration position in diagnostic camera pixels, where the multiprobe is visible on the diagnostic
-        # camera and the mppc detector.
-        FAV_POS_ACTIVE: {"x": 750, "y": 550}
+        # x and y: Factory calibration position in diagnostic camera pixels, where the multiprobe is visible on
+        # the diagnostic camera and the mppc detector.
+        # z: Simulated stage position where the camera is in focus; choose different from stage metadata
+        # to better simulate the autofocusing procedure. Note: ONLY needed for simulator.
+        FAV_POS_ACTIVE: {"x": 750,  # [px_dc] good multiprobe position
+                         "y": 550,  # [px_dc] good multiprobe position
+                         "z": -43.e-6}  # [m] position of stage where the multiprobe is displayed in focus
     },
 }
 
 
```

Comment:
is two empty lines at the end of yaml files the standard? I can't really find anything on it online, so why 2 and not 1 empty line?